src/structures/player.ts

```diff
@@ -1,30 +1,30 @@
 export default class Player {
     /** The Discord tag of the player */
-    private tag: string;
+    public readonly name: string;
 
     /** The Discord user ID of the player */
-    private id: string;
+    public readonly id: string;
+
+    /** The player's avatar URL */
+    private readonly avatarURL: string;
 
     /** The player's current score */
     private score: number;
 
-    /** The player's avatar URL */
-    private avatarURL: string;
-
     /** The player's EXP gain */
     private expGain: number;
 
     constructor(tag: string, id: string, avatarURL: string, points: number) {
-        this.tag = tag;
+        this.name = tag;
         this.id = id;
         this.score = points;
         this.avatarURL = avatarURL;
         this.expGain = 0;
     }
 
     /** @returns the player's Discord tag  */
-    getTag(): string {
-        return this.tag;
+    getName(): string {
```

Comment:
Why rename this

Reply:
Since the class Team extends from this class, it doesn't make as much sense to call .getTag() on a team. Up to you though, I can change it back